interface/new/new_comprehensive.php

```diff
@@ -416,24 +416,25 @@ function searchme() {
 <form action='new_comprehensive_save.php' name='demographics_form' id="DEM"  method='post' onsubmit='return submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,"DEM",constraints)'>
 
     <div class="container-fluid">
-        <div class="row">
-            <div class="col-xs-12">
-                <div class="page-header">
-                    <h1><?php echo xlt('Search or Add Patient');?></h1>
+        <div class="page-header">
+            <div class="row">
+                <div class="col-xs-12 col-sm-6 col-md-4">
+
+                    <h2><?php echo xlt('Search or Add Patient');?></h2>
                 </div>
-            </div>
-            <div class="col-xs-12">
-                <div class="btn-group">
-                    <?php if ($WITH_SEARCH) { ?>
-                        <button type="button" class="btn btn-default btn-search" id="search" value="<?php echo xla('Search'); ?>">
-                            <?php echo xlt('Search'); ?>
+
+                <div class="col-xs-12 col-sm-6 col-md-4">
```

Comment:
This needs to go below the form, preferably inline with the left-most input field, though that may not be super feasible in this case.